From 0dd04a25075c4e7c62996aac76e8b4ea3f6da7f3 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Wed, 26 Dec 2018 13:19:45 +0100
Subject: [PATCH 1/1] efi_loader: avoid unnecessary pointer to long conversion

debug() support supports %p to print pointers.

The debug message is unique. So there is not need to write a possibly
distracting line number.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 cmd/bootefi.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/cmd/bootefi.c b/cmd/bootefi.c
index 9f4c871ba3..9ed15ec083 100644
--- a/cmd/bootefi.c
+++ b/cmd/bootefi.c
@@ -404,8 +404,7 @@ static efi_status_t do_bootefi_exec(void *efi,
 		"{ro,boot}(blob)0000000000000000");
 
 	/* Call our payload! */
-	debug("%s:%d Jumping to 0x%lx\n", __func__, __LINE__,
-	      (long)image_obj->entry);
+	debug("%s: Jumping to 0x%p\n", __func__, image_obj->entry);
 
 	if (setjmp(&image_obj->exit_jmp)) {
 		ret = image_obj->exit_status;
-- 
2.19.2

